PD-79 Consent tracking

edwardupton opened the pull request.

lyykfi commented on src/common/helpers/getTrackingConsent.ts:
```diff
@@ -5,18 +5,29 @@ declare let window: CustomWindow;
+				},
+			],
+			(error: any) => {
+				if (error) return resolve(false);
```
wdyt to add tests here?

edwardupton replied:
I already did:
https://github.com/littledata/shopify-tracker/pull/260/files#diff-115a49c1b0959ee1ffc1bb23ca79d3d6d9d6523fea40329c69437ccbbc071f9fR23

lyykfi commented:
could you rebase by staging please to fix problems with tests?

lyykfi commented:
@edwardupton have you regenerated dist/*.js files?

edwardupton commented:
No, but I updated from master with /dist folder changes. I've reverted that

On Wed, Jun 23, 2021 at 10:21 AM Andrey ***@***.***> wrote:

> @edwardupton <https://github.com/edwardupton> have you regenerated
> dist/*.js files?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/littledata/shopify-tracker/pull/260#issuecomment-866676458>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAV6OKISEB6X2GGYMAXWNWLTUGRP3ANCNFSM46545O6Q>
> .
>

lyykfi commented:
Thank. I just though that there was a potential problem with the /dist folder.
approved
